Documentation/ceph-upgrade.md

```diff
@@ -231,121 +234,72 @@ mkdir backups
-automatically.
+The largest portion of the upgrade is triggered when the operator's image is updated to `v1.0.x`.
 ```sh
-kubectl -n $ROOK_SYSTEM_NAMESPACE set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v0.9.0
+kubectl -n $ROOK_SYSTEM_NAMESPACE set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.0.0
 ```
 
-Watch now in amazement as the Ceph MONs, MGR, OSDs, RGWs, and MDSes are terminated and replaced with
-updated versions in sequence. The cluster may be offline very briefly as MONs update, and the Ceph
-Filesystem may fall offline a few times while the MDSes are upgrading. This is normal. Continue on
-to the next upgrade step while the update is commencing.
+Watch now in amazement as the Ceph mons, mgrs, OSDs, rbd-mirrors, mdses and rgws are terminated and
+replaced with updated versions in sequence. The cluster may be offline very briefly as mons update,
+and the Ceph Filesystem may fall offline a few times while the mdses are upgrading. This is normal.
+Continue on to the next upgrade step while the update is commencing.
+
+### 4. Wait for the upgrade to complete
+Before moving on, the Ceph cluster's core (RADOS) components should be fully updated.
 
-### 4. Delete the old MGR replicaset
-After the MONs have updated, the MGR will update, and because of the new MGR service account with
-fewer decreased privileges, the old MGR replica set must be deleted manually. Once you see two MGR
-replica sets, delete the old one using the Rook v0.8 image.
 ```sh
-kubectl -n $ROOK_NAMESPACE get rs -l app=rook-ceph-mgr \
-  -o custom-columns='name:.metadata.name,image:.spec.template.spec.containers[0].image'
-# Sample output:
-#  rook-ceph-mgr-a-deletethis   rook/ceph:v0.8.3
-#  rook-ceph-mgr-a-dontdelete   ceph/ceph:v12.2.9-2018102
-kubectl -n $ROOK_NAMESPACE delete rs 'rook-ceph-mgr-a-deletethis'  # edit name to match your output
+watch --color --exec kubectl -n $ROOK_NAMESPACE get deployments -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.labels.rook-version}{"\n"}{end}'
 ```
 
-### 5. Wait for the upgrade to complete
-Before moving on, the cluster's main components should be fully updated. A telltale indication that
-the upgrade is nearly finished is that all the osd pods will have been renamed slightly. An example
-of an OSD pod name from v0.8 is `rook-ceph-osd-id-0-6795dff4bb-d7pz9`. This will have changed to
-something like `rook-ceph-osd-0-74fcfd9c5b-58gd8` after the upgrade where the `id-` has been dropped
-between `osd` and the OSD ID.
+As an example, this cluster is midway through updating the OSDs from 0.9 to 1.0. When all
+deployments report `1/1/1` availability and `rook-version=v1.0.0`, the Ceph cluster's core
+components are fully updated.
 ```sh
-watch -c kubectl -n $ROOK_NAMESPACE get pods
+Every 2.0s: kubectl -n rook-ceph get deployment -o j...
```

Comment:
Nice output! Would it be too long to spell out the `req/upd/avl`?

Reply:
I did that initially, and it seemed a little too lengthy given the output is usually just `1/1/1`. I could change if you think it might confuse anyone.

Reply:
If you spell it out in this doc I'm sure it should be sufficient. At least I was confused without it being spelled out.